README.md

```diff
@@ -21,23 +21,23 @@ It also generates TypeScript types, JSON maps of the generated code-points, allo
 
 ### Install
 
-```
+```bash
 npm install -g fantasticon
 ```
 
 ## Use
 
 ### Quick usage
 
-```
+```bash
 fantasticon my-icons -o icon-dist
 ```
 
 ### Command-line
 
-**Note:** Not all options can be specified through the command line - for `formatOptions`, `pathOptions` and `templates` use a [configuration file](#configuration-file) or the javaScript API.
+**Note:** Not all options can be specified through the command line - for `formatOptions`, `pathOptions` and `templates` use a [configuration file](#configuration-file) or the JavaScript API.
 
-```bash
```

Comment:
Removed this because the highlighting didn't make a lot of sense to me:

![image](https://user-images.githubusercontent.com/349621/103093881-72840400-4604-11eb-9e74-3af860c377ae.png)

But if it was intentional and preferred I can revert this :)

Reply:
Nah that doesn't really make sense I guess, so better this way :)